api/meter_test.go

```diff
@@ -1074,6 +1076,11 @@ func (f *fakeBackend) TodaysUsage() (map[string]RelayCounts, error) {
 	return f.todaysUsage, nil
 }
 
+func (f *fakeBackend) TodaysLatency() (map[string][]Latency, error) {
+	f.todaysLatencyCalls++
+	return f.todaysLatency, nil
+}
```

Comment:
Shouldn't the mock allow testing the error case? i.e. when the backend service returns an error?
This should also be covered by the unit tests.

Reply:
Added the latency funcs to the unit tests, including testing backend errors.